app/export.go

```
@@ -91,7 +100,7 @@ func (app *EthermintApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAd
 	for _, addr := range jailAllowedAddrs {
 		_, err := sdk.ValAddressFromBech32(addr)
 		if err != nil {
-			return err
+			log.Fatal(err)
```

@dudong2 
Most of error handlings here are changed to panic instead of return error.
Is there any special reason for those changes?

The original handling flow (return err --> get err from CLI command) seems good to me.